### ISSUE TITLE
Network error everytime I try to download schemas

### ISSUE THREAD

**Issue by diaslag13** [Frodo45127/rpfm#115]
**Describe your problem**
When I try to launch rfpm 2.0 it says "No local schemas found. Do you want to download the lastest ones?
NOTE: Schemas are needed for opening tables, locs and other PackedFiles. No schemas means you cannot edit tables." I clicked update then it says "Downloading updates, don't close this window...
This may take a while." After 10 seconds it will always says that "There has been a network related error. Please, try again later"

Can I turn off the update nag everytime I launch rfpm 1.6? rfpm 2.0 is useless to me

**Extra info?**
 rfpm 2.0

**Comment by GrouchPotato** [comment]
+1 With this problem.

This is on windows 10. I've tried adding explicit permissions to the firewall config, but that didn't help.

Is there a direct download like for the schema files?

**Comment by GrouchPotato** [comment]
I just checked my AppData/.........../config/schema folder and realised it'd downloaded some of the schemas but not all. Hitting it with the retry stick solved the problem. I guess the issue is in the downloader's fault tolerance, which given my internet is rubbish would add up.

**tldr; The workaround is to keep hitting it with the retry stick.**

**Comment by Frodo45127** [comment]
Yeah, as GrouchPotato said, just hit retry. The downloader is not very reliable, so sometimes it takes multiple tries to get it working. But it works in the end.

**Comment by diaslag13** [comment]
Thanks for help guys it finally  worked now. I just click "about" then "Check schema updates". Basically just manually update the schema files. I hope the update bug could be patced in the next version.

**Comment by diaslag13** [comment]
Thanks for help guys it finally  worked now. I just click "about" then "Check schema updates". Basically just manually update the schema files. I hope the update bug could be patced in th
